src/index.js

```diff
@@ -0,0 +1,126 @@
+"use strict";
+
+const tempBox = document.getElementById("TempBox");
+let tempValue=  Number(tempBox.textContent);
+
+const increaseTemp = () => {
+    tempValue += 1;
+    changeTemp(tempValue)
+    tempBox.textContent = tempValue;
+    changePlants();
+}
+
+const decreaseTemp = () => {
+    tempValue -= 1;
+    changeTemp(tempValue)
+    tempBox.textContent = tempValue;
+    changePlants();
+}
+
+const changeTemp = (tempValue) => {
+    
+    let color = "white";
+    if ( tempValue >= 80) {
+        color = "red";
+    } else if (tempValue >= 70) {
+        color = "orange";
+    } else if (tempValue >= 60) {
+        color = "yellow";
+    } else if (tempValue >= 50) {
+        color = "green";
+    } else {
+        color = "teal"
+    }
+    tempBox.classList = color;
+}
+
+const changeSky = () => {
+    // gets the value chosen in dropdown
+    const inputSky = document.getElementById("sky-select").value;
+    // gets the box where the sky is going to go
+    const skyContainer = document.getElementById("sky");
+    let sky = "🌤";
+    let skypic = "";
+    if (inputSky === "Cloudy") {
+        sky = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+        skypic = "cloudy";
+    } else if (inputSky === "Sunny") {
+        sky = "☁️     ☁️   ☁️ ☀️ ☁️  ☁️";
+        skypic = "sunny";
+    } else if (inputSky === "Rainy") {
+        sky = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧";
+        skypic = "rainy";
+    } else if (inputSky === "Snowy") {
+        sky = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+        skypic = "snowy";
+    }
+    skyContainer.textContent = sky;
+    const gardenContent = document.getElementById("weather-garden");
+    // what is this doing???
+    gardenContent.classList = skypic;
```

Comment:
this is grabbing the property in your CSS file and changing the color properties value.